### PR TITLE
Add of three stats

### PR DESCRIPTION
**Pull request by Maxime-Thibaud**
I have added 3 progress bars: 2 new stats and the result of the comparison.

### CODE REVIEW

File: gitvs/src/components/PublicGists.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  const publicGists1 = props.firstUserPublic_Gists
+  const publicGists2 = props.secondUserPublic_Gists
+  const publicGistsTotal = (publicGists1 + publicGists2)
+  const publicGistsPercent = (publicGists1 / publicGistsTotal) * 100
+  const publicGistsDecimal = Math.round(publicGistsPercent * 1) / 1
+  const publicGistsSecondUser = Math.round((100 - publicGistsPercent) * 1) / 1
+
+
+  const classes = useStyles()
+  const [completed, setCompleted] = React.useState(0)
```
Comment (erabin05) [inline gitvs/src/components/PublicGists.js]: Hook "useState()" must be at the top of your component

File: gitvs/src/components/ResultOfFight.js [hunk 2]
```diff
@@ -1,21 +1,38 @@
...
+  const publicGists2 = props.secondUserPublic_Gists
+  const publicGistsTotal = (publicGists1 + publicGists2)
+  const publicGistsPercent = (publicGists1 / publicGistsTotal) * 100
+  const publicGistsDecimal = Math.round(publicGistsPercent * 1) / 1
+  const publicGistsSecondUser = Math.round((100 - publicGistsPercent) * 1) / 1
+
+  const resultUser1 = (percentDecimal + followerDecimal + publicGistsDecimal) / 3
+  const resultUser2 = (percentSecondUser + followerSecondUser + publicGistsSecondUser) / 3
   const classes = useStyles()
   const [completed, setCompleted] = React.useState(0)
```
Comment (erabin05) [inline gitvs/src/components/ResultOfFight.js:37]: same : usState on top

File: gitvs/src/components/Comparison.js [hunk 3]
```diff
@@ -2,36 +2,76 @@
...
+    }
+  }
 
-const Comparison = (props) => (
-  <div id="comparison">
-    <div className="VS-phone">
-      <figure>
+  resultAppear() {
+    this.setState({ isResultIsVisible: true })
+  }
```
Comment (erabin05) [inline gitvs/src/components/Comparison.js:23]: Don't you need to bind this method ?

**Review by erabin05 — CHANGES_REQUESTED** [review]
- A method to bind
- It is recommended to pu useState() at the top of your component

**Comment by Maxime-Thibaud** [comment]
I have made the requested changes

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**